Bukkit/src/main/java/com/plotsquared/bukkit/listener/BlockEventListener.java

```diff
@@ -1068,6 +1083,16 @@ public void onBlockIgnite(BlockIgniteEvent event) {
         Plot plot = area.getOwnedPlot(location1);
         if (player != null) {
             BukkitPlayer pp = BukkitUtil.adapt(player);
+            if (!area.buildRangeContainsY(location1.getY()) && !Permissions
```

Comment:
Any objections about refactoring this? This is now the sixth time we reuse this message part, this could live on as protected method helper.

Reply:
It's sent mostly in BlockEventListener but a protected static method in PlayerEventListener might make more sense since it's specifically for players?

Reply:
Based on a quick look over "duplicated methods" in IJ earlier, this is used here and one time in the BlockEventListener117 class.
+1 for adding it to the PlayerEventListener.